cartridges/openshift-origin-cartridge-jbossews/bin/control

```diff
@@ -87,6 +87,15 @@ function ishttpup() {
     return 1
 }
 
+# This function is here to mimmic the EAP cartriges design: 
+# Due to the lack of a provided mgmt interface, the options to check for 
+#  deployment activation is limited. Thus this fucntion uses an ENV VAR to 
+#  allow you configure a sleep time (in seconds), giving your deployments 
+#  time to start before marking the gear as active. 
+function waitondeployments() {
+    sleep ${OPENSHIFT_JBOSSEWS_START_DELAY:-2}
```

Comment:
Why was the default set to wait 2 seconds? Is it necessary to wait 2 seconds on every start of the application?